blog/docs/events/virtual-office-hour-12-2022.md

```diff
@@ -0,0 +1,46 @@
+# First 🌱 Knative Virtual Office Hour
+
+**Date: 2022-12-01**
+
+**Authors: [Alek Slominski](https://aslom.net/), [Evan Anderson](https://off-by-one.dev/), [Lance Ball](https://twitter.com/lanceball), and Paul Schweigert**
+
+We think that there should be an easy way for new Knative users and contributors to get started with our project. Holding virtual office hours for questions and help with using or contributing to Knative should help to get started with using code and contributing.
+
+During the recent KubeCon NA 2022 in Detroit we had Knative Day, Maintainer Meetup, and ContribFest that showed there are people interested in using or contributing to Knative. However, today we do not have any ongoing online meetings for people to just show up. They can join Slack but that is asynchronous and requires joining yet another Slack, then reading channels, and figuring where to ask your questions. Driving your own engagement can be a barrier to participating in the community.
+
+Therefore, we are organizing the first Knative Virtual Office Hour in 2022.
+
+**When:** [December 9 (Friday) 11:30 AM US Eastern](https://dateful.com/convert/utc?t=430pm&d=2022-12-09)
+**How to join:** Zoom link will be posted in Slack and knative-user mailing list
+
+**Who should join?**
+
+- You are curious about Knative and you are wondering where to start
+- You have great idea for Knative but you do not know where or how to share it
+- You want to get to know how Knative works inside and want to contribute to open source project
+
+**Possible topics for Contib fest:**
+
+- See [slides form KubeCon about Knative ContribFest](https://docs.google.com/presentation/d/1hnPHtbjLs3KMe-N62NhQeVtjH-p-obyI/edit?usp=sharing&ouid=112051875307901839973&rtpof=true&sd=true)
+- Browse good first issues:
+  - [client](https://github.com/knative/client/issues?q=is%3Aissue+is%3Aopen+label%3Akind%2Fgood-first-issue)
+  - [docs](https://github.com/knative/docs/issues?q=is%3Aissue+is%3Aopen+label%3Akind%2Fgood-first-issue)
+  - [eventing](https://github.com/knative/eventing/issues?q=is%3Aissue+is%3Aopen+label%3Akind%2Fgood-first-issue)
+  - [func](https://github.com/knative/func/issues?q=is%3Aissue+is%3Aopen+label%3Akind%2Fgood-first-issue)
+  - [serving](https://github.com/knative/serving/issues?q=is%3Aissue+is%3Aopen+label%3Akind%2Fgood-first-issue)
+- Browse eventing sources in knative-sandbox to see what issues could be fixed of features added such as
+  - [eventing-couchdb](https://github.com/knative-sandbox/eventing-couchdb)
+  - [eventing-github](https://github.com/knative-sandbox/eventing-couchdb)
+  - [eventing-gitlab](https://github.com/knative-sandbox/eventing-couchdb)
+  - [eventing-kogito](https://github.com/knative-sandbox/eventing-couchdb)
+  - [eventing-redis](https://github.com/knative-sandbox/eventing-couchdb)
+  - Other [eventing sources and sandbox projects](https://github.com/knative-sandbox/)
```

Comment:
```suggestion
```

I'm not sure we want to link the Contrib fest part here past the slides.

Instead, I added a couple more items for "who should join".

Reply:
OK with me - we can see if there is need for virtual contrib fest in future.